vogue/commands/load/application_tag.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import click
+import json
+from vogue.load.application_tag import load_aplication_tags
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command("apptag", short_help="Reads json with application tag info.")
+@click.option('-a', '--application-tags', required=True, help='json formatted application tags')
```

Comment:
If this is the only required option, make it an argument.
This needs to be a string :) (not a file as previously stated)

```suggestion
@click.('application-tags', help='json formatted application tags')
```

Reply:
I cant get it to work as argument. tried this to. 
@click.argument("application_tags", type=click.STRING)
Ill open a separate pr to solve this.

Reply:
yes that should work.
```@click.argument("application_tags", help='json formatted application tags')```

It's a bad habit to fix things in other PRs ;)